src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunMinimalBWAMEMTest.java

```diff
@@ -0,0 +1,106 @@
+    private static final Path bwaPath;
+    static {
+        if (System.getProperty("user.name").equalsIgnoreCase("shuang")) {
+            bwaPath = Paths.get("/usr/local/bin/bwa");
+        } else {
+            bwaPath = Paths.get("/home/travis/bwaApache2/bwa");
+        }
+    }
+
+    @Test(groups="sv")
+    public void testSeparate() throws IOException {
+
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        final File samOutput = boilerPlate(args);
+
+        // input arguments
+        final File input = new File(TEST_DATA_DIR, "input_1.fastq");
+        args.add("-" + StandardArgumentDefinitions.INPUT_SHORT_NAME);
+        args.add(input.getAbsolutePath());
+        final File secondInput = new File(TEST_DATA_DIR, "input_2.fastq");
+        args.add("-" + "I2");
+        args.add(secondInput.getAbsolutePath());
+
+        this.runCommandLine(args.getArgsArray());
+
+        Assert.assertEquals(getSamRecords(expectedSam), getSamRecords(samOutput));
+    }
+
+    @Test(groups="sv")
+    public void testInterLeaved() throws IOException {
+
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        final File samOutput = boilerPlate(args);
+
+        // input arguments
+        args.add("-p");
+        final File input = new File(TEST_DATA_DIR, "interleaved.fastq");
+        args.add("-" + StandardArgumentDefinitions.INPUT_SHORT_NAME);
+        args.add(input.getAbsolutePath());
+
+        this.runCommandLine(args.getArgsArray());
+
+        Assert.assertEquals(getSamRecords(expectedSam), getSamRecords(samOutput));
+    }
+
+    private static File boilerPlate(final ArgumentsBuilder args) throws IOException{
+
+        args.add("-" + "bwaPath");
+        args.add(bwaPath.toString());
+
+        final File wkDir = BaseTest.createTempDir("dummy");
+        args.add("-" + "outDir");
+        args.add(wkDir.getAbsolutePath());
+
+        final File output = new File(wkDir, "test.sam");
+        output.createNewFile();
+        args.add("-" + StandardArgumentDefinitions.OUTPUT_SHORT_NAME);
+        args.add(output.getName());
+
+        final File REF = ref.toFile();
+        args.add("-" + StandardArgumentDefinitions.REFERENCE_SHORT_NAME);
+        args.add(REF.getAbsolutePath());
+
+        return output;
+    }
+
+    // utility function to extract just alignment records
+    // header information depends on environment in which the test is run and expected sam is generated, so is un-reproducible
+    private static List<String> getSamRecords (final File samFile) throws IOException{
+        final List<String> samRecords = new ArrayList<>();
+        try(final BufferedReader reader = new BufferedReader(new FileReader(samFile))) {
+            String line;
+            while ( (line = reader.readLine()) != null) {
+                if(!line.startsWith("@")){
+                    samRecords.add(line);
+                }
+            }
+        }catch(final IOException e) {
```

Comment:
dont catch IOException if you're going to throw IOException. Just delete the try/catch